### PR TITLE
Wrote unit tests for 'StepperControlButtons' component #422

### PR DESCRIPTION
**Pull request by dmytro07**
Added unit tests for 'StepperControlButtons' component
tests

### CODE REVIEW

**Comment by AndriyLymych** [comment]
please, link your issue 
image

File: src/components/buttons/stepper-control-buttons/tests/stepper-control-buttons.test.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  afterEach(() => {
+    component.unmount();
+    mockHandleBack.mockClear();
+    mockHandleNext.mockClear();
+    useSelectorMock.mockClear();
+  });
+
+  it('Should match snapshot', () => {
+    expect(toJson(component)).toMatchSnapshot();
```
Comment (Haytham1337) [inline src/components/buttons/stepper-control-buttons/tests/stepper-control-buttons.test.js:50]: remove Snapshot test

File: src/components/buttons/stepper-control-buttons/tests/stepper-control-buttons.test.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+const { NEXT, CREATE_PRODUCT, BACK } = productsTranslations;
+const mockStore = {
+  Products: {
+    loading: true
+  }
+};
+
+describe('Stepper control buttons tests', () => {
+  const useSelectorMock = jest.spyOn(reactRedux, 'useSelector');
+  const mockHandleNext = jest.fn(noop());
```
Comment (Kiyooooko) [inline src/components/buttons/stepper-control-buttons/tests/stepper-control-buttons.test.js:25]: please remove () after noon

File: src/components/buttons/stepper-control-buttons/tests/stepper-control-buttons.test.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  const mockHandleBack = jest.fn(noop());
```
Comment (Kiyooooko) [inline src/components/buttons/stepper-control-buttons/tests/stepper-control-buttons.test.js:26]: same

**Comment by Kiyooooko** [comment]
please delete _snapshots_ directory after deleting function .toMatchSnapshot()

**Review by Haytham1337 — APPROVED** [review]
Well done!

**Comment by Kiyooooko** [comment]
nice job! 🔥

**Review by AndriyLymych — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Pull request merged**